File: java_work/ex09/src/Ex01.java
class A{
    int num = 11;
    public void doA(){
        System.out.println("아무렇게 만들기");
    }
}
/*
ㄴ 클래스 정의 원래는 파일 나누는게 국룰
 */
public class Ex01 {

    public static void main(String[] args) {

        int num = 10;//여러개 선언 가능
        int num2 = 20;
        int num3 = 30;

        // 선언하는 순간 메모리에 적재된다.
        // 메모리 안에 클래스를 적재할때 new를 쓴다
        //
        A a1 = new A();
        A a2 = new A();
        A a3 = new A();

        System.out.println(a1.num);
        a1.doA();
        a1.num = 30;
        System.out.println("a1.num= "+a1.num);


        System.out.println("a2.num= "+a2.num);
        a2.doA();
    }
}

File: html_work/15/ex01.js
//let 은 변경 가능한 변수
//const 는 변경불가능한 상수

let aa = document.getElementById("btn-min");
const bb = document.getElementById("btn-max");
const printDiv = document.getElementById("print-div");

const n1 = document.getElementById("num1");
const n2 = document.getElementById("num2");
const n3 = document.getElementById("num3");
const n4 = document.getElementById("num4");
const n5 = document.getElementById("num5");

function min(){
 //console.log(n1.value);
 const arr =[n1.value,n2.value,n3.value,n4.value,n5.value,];
 let min = n1.value
 for(let index = 0; index <arr.length ; index++){
  const element = arr[index];
  if(Number(min)>Number(element))
    min = element;
 }
 printDiv.textContent = '최솟값 = '+min;
}

function max(){
  const arr =[n1.value,n2.value,n3.value,n4.value,n5.value,];
  let max = n1.value
  for(const key of arr){
   if(Number(max) < Number(key));
     max = key;
  }
 printDiv.textContent = '최대값 = '+ max;
}

// function max(){
//   alert('max함수 호출')
// }

aa.addEventListener('click',min);
bb.addEventListener('click',max);

printDiv.textContent = "안녕하세요 내용을 넣는거에요...";


// const cc = 10;
// let dd = 10;

// // cc=20; 이캐뿌면 에러
//  dd=30; //이건 OK

// console.log(dd);

// for(let i = 0; i<10; i++){
//   console.log(i);
// }



